### PR TITLE
[14.0][MIG] purchase_triple_discount

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration

- Changed some test for new value on field price_total, include taxes
- Remove models no necessary for V14

@ForgeFlow

### CODE REVIEW

File: purchase_triple_discount/models/purchase_order.py [hunk 1]
```diff
@@ -51,3 +49,25 @@ def _apply_value_from_seller(self, seller):
...
+    @api.model
+    def _prepare_purchase_order_line_from_procurement(
+        self, product_id, product_qty, product_uom, company_id, values, po
+    ):
+        res = super(
+            PurchaseOrderLine, self
+        )._prepare_purchase_order_line_from_procurement(
+            product_id, product_qty, product_uom, company_id, values, po
+        )
+        return res
```
Comment (legalsylvain) [inline purchase_triple_discount/models/purchase_order.py]: what is the interest of there lines ?

File: purchase_triple_discount/models/purchase_order.py [hunk 2]
```diff
@@ -51,3 +49,25 @@ def _apply_value_from_seller(self, seller):
...
+
+    def _prepare_account_move_line(self, move=False):
+        self.ensure_one()
+        res = super(PurchaseOrderLine, self)._prepare_account_move_line(move)
```
Comment (legalsylvain) [inline purchase_triple_discount/models/purchase_order.py]: ``         res = super(PurchaseOrderLine, self)._prepare_account_move_line(move)``
->
``         res = super()._prepare_account_move_line(move)``

**Review by legalsylvain — COMMENTED** [review]
some minors remarks. 
otherwise, LGTM. Thanks for porting this module.
Tested on runboat. works like a charm.

regards.

note : travis fail due to an error in ``purchase_quick``. don't understand why. 
@kevinkhao : any idea ? (ref https://github.com/OCA/purchase-workflow/pull/1213)

File: purchase_triple_discount/models/purchase_order.py (hunk 1 above)
Comment (ChrisOForgeFlow) [reply purchase_triple_discount/models/purchase_order.py]: When I reviewed version 12, there's code related with procurement, but I don't need on this migration

File: purchase_triple_discount/models/purchase_order.py (hunk 2 above)
Comment (ChrisOForgeFlow) [reply purchase_triple_discount/models/purchase_order.py]: Done

**Review by dreispt — APPROVED** [review]

**Review by legalsylvain — APPROVED** [review]
Lgtm.
- Could you add me as maintainers ?
- travis is still red.
Regards.

**Pull request merged**